Content.Shared/Chemistry/Components/InjectorComponent.cs

```diff
@@ -81,6 +81,12 @@ public sealed partial class InjectorComponent : Component
     [AutoNetworkedField]
     [DataField]
     public InjectorToggleMode ToggleState = InjectorToggleMode.Draw;
+
+    [DataField]
+    public bool NeedHand = true;
```

Comment:
While I do understand it's a pretty drop in place replacement for the doafter variables, please leave summary comments about these, preferably explaining that they affect doafters and also explaining when you would ever want/need it to be false